docs/Makefile

```diff
@@ -11,7 +11,7 @@ docs: $(patsubst %.md,%,$(wildcard *.md))
 .PHONY: install
 install:
 	install -d ${DESTDIR}/${MANDIR}/man5
-	install -m 0644 *.5 ${DESTDIR}/${MANDIR}/man5
+	install -m 0644 .*.5 *.5 ${DESTDIR}/${MANDIR}/man5
```

Comment:
OK, I'll show my Makefile syntax ignorance.  What does this change accomplish?

Reply:
I had to ask people if there was a better way, but essentially this isn't a Makefile issue, it's a globbing issue. `*` doesn't match hidden/dotted files so we add the explicit dot to catch it.

<pre> 
λ docs morten/move-manpages» ls *.5*
Containerfile.5.md  containerignore.5.md  containers.conf.5.md  containers-mounts.conf.5.md
λ docs morten/move-manpages» ls .*.5*
.containerignore.5
</pre>